src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -106,18 +106,18 @@ def __init__(self, volclient, vol_spec):
         # reporting has already been initiated by calling RTimer.is_alive().
         self.update_task = RTimer(1, self._update_progress_bars)
 
+        # progress event ID for ongoing clone jobs
+        self.on_pev_id: Optional[str] = 'mgr-vol-ongoing-clones'
```

Comment:
does it need to be Optional if we're not going to assign None?

Reply:
Yes, else mypy throws a warning otherwise which causes `make check` to fail